pennylane/operation.py

```diff
@@ -636,16 +646,32 @@ def get_parameter_shift(self, idx):
         """
         # get the gradient recipe for this parameter
         recipe = self.grad_recipe[idx]
-        multiplier, shift = (0.5, np.pi / 2) if recipe is None else recipe
 
-        # internal multiplier in the Variable
-        var_mult = self.data[idx].mult
+        # Default values
+        multiplier = 0.5 / np.sin(shift)
+        a = 1
+
+        # We set the default recipe following:
+        # ∂f(x) = c1*f(a1*x+s1) + c2*f(a2*x+s2)
+        # where we express a positive and a negative shift by default
+        default_param_shift = [[multiplier, a, shift], [-multiplier, a, -shift]]
+        param_shift = default_param_shift if recipe is None else recipe
 
-        multiplier *= var_mult
-        if var_mult != 0:
-            # zero multiplier means the shift is unimportant
-            shift /= var_mult
-        return multiplier, shift
+        if hasattr(self.data[idx], "mult"):
+            # Parameter is a variable, we are in non-tape mode
+            # Need to use the internal multiplier in the Variable to update the
+            # multiplier and the shift
+            var_mult = self.data[idx].mult
+
+            for elem in param_shift:
+
+                # Update the multiplier
+                elem[0] *= var_mult
+                if var_mult != 0:
+                    # Update the shift
+                    # zero multiplier means the shift is unimportant
+                    elem[2] /= var_mult
```

Comment:
I think I'm missing this part, why divide and why the `elem[2]`?

Reply:
This is related to the slightly obscured way that gradient rules are implemented in non-tape mode. In non-tape mode, the multiplier in the shift rule is always absorbed into the `Variable.mult` attribute.

Regarding the division, this is because we need to transform from `f(mult*x + s)` to `f(mult*(x+s/mult))`, again due to an implementation detail of the non-tape mode shift rule.

Note that this logic remains the same as previously; the line `shift /= var_mult` is directly equivalent; the only difference is that the shift value is now the 2nd element of each param-shift list.

Reply:
`f(mult*(x+s/mult))`
^ I get it! Thanks!